src/casus/saveCasus.js

```diff
@@ -5,9 +5,12 @@ import getTankForCasus from '../globalComponents/getTankForCasus.js';
 import ContainerBlock from '../casus/blocks/ContainerBlock.js';
 import { toast } from 'react-toastify';
 
-function saveCasus(casusCode: ContainerBlock): void {
+function saveCasus(casusCode: ContainerBlock, tankID: ?string, onLoad:() => void): void {
 	
-	const tankToEditID: string = getTankForCasus();
+	let tankToEditID: ?string = tankID;
```

Comment:
You can clean this up to be: `const tankToEditID = tankID ?? getTankForCasus();`